`make --trace` broken

I am trying to figure out some build issues, and usually `make --trace` is a great tool for debugging. However, not so with ssreflect -- it does not build with `--trace`:
```
$ make --trace
Makefile:24: target 'all' does not exist
[ -e Makefile.coq ] ||  (echo "Generating Makefile.coq for Coq v8.7 with COQBIN=/home/r/.opam/coq-8.7/bin//"; if [ "$OS" = "Windows_NT" ]; then LN=cp; else LN="ln -sfr"; fi; MLLIB=; EXTRA=; case v8.7 in v8.5*|v8.4*) $LN ./plugin/v8.7/ssrmatching.mli .; $LN ./plugin/v8.7/ssrmatching.ml4 .; $LN ./plugin/v8.7/ssrmatching.v .; $LN ./plugin/v8.7/ssreflect_plugin.mllib .; $LN ./plugin/v8.7/ssreflect.ml4 .; $LN ./plugin/v8.7/ssrbool.v ./; $LN ./plugin/v8.7/ssrfun.v ./; $LN ./plugin/v8.7/ssreflect.v ./; MLLIB=ssreflect_plugin.mllib; EXTRA="ssrmatching.mli ssrmatching.ml4 ssrmatching.v ssreflect.ml4"; ;; v8.6*) $LN ./plugin/v8.7/ssreflect_plugin.mlpack .; $LN ./plugin/v8.7/ssreflect.ml4 .; $LN ./plugin/v8.7/ssrbool.v ./; $LN ./plugin/v8.7/ssrfun.v ./; $LN ./plugin/v8.7/ssreflect.v ./; MLLIB=ssreflect_plugin.mlpack; EXTRA="ssreflect.ml4"; ;; esac; /home/r/.opam/coq-8.7/bin//coq_makefile -f Make $MLLIB $EXTRA -o Makefile.coq)
Generating Makefile.coq for Coq v8.7 with COQBIN=/home/r/.opam/coq-8.7/bin//
# Override COQDEP to find only the "right" copy .ml files
MAKEFLAGS= --trace make --no-print-directory \
        -f Makefile.coq all \
        COQDEP='/home/r/.opam/coq-8.7/bin//coqdep -exclude-dir plugin -c'
/bin/sh: 1: --trace: not found
Makefile:24: recipe for target 'all' failed
make: *** [all] Error 127
```
This is in a clean git clone of this repository, in the `mathcomp/ssreflect` folder, with Coq 8.7 beta 2.

The result with Coq 8.6.1 is pretty much the same:
```
$ make --trace
Makefile:24: target 'all' does not exist
[ -e Makefile.coq ] ||  (echo "Generating Makefile.coq for Coq v8.6 with COQBIN=/home/r/.opam/coq-8.6/bin//"; if [ "$OS" = "Windows_NT" ]; then LN=cp; else LN="ln -sfr"; fi; MLLIB=; EXTRA=; case v8.6 in v8.5*|v8.4*) $LN ./plugin/v8.6/ssrmatching.mli .; $LN ./plugin/v8.6/ssrmatching.ml4 .; $LN ./plugin/v8.6/ssrmatching.v .; $LN ./plugin/v8.6/ssreflect_plugin.mllib .; $LN ./plugin/v8.6/ssreflect.ml4 .; $LN ./plugin/v8.6/ssrbool.v ./; $LN ./plugin/v8.6/ssrfun.v ./; $LN ./plugin/v8.6/ssreflect.v ./; MLLIB=ssreflect_plugin.mllib; EXTRA="ssrmatching.mli ssrmatching.ml4 ssrmatching.v ssreflect.ml4"; ;; v8.6*) $LN ./plugin/v8.6/ssreflect_plugin.mlpack .; $LN ./plugin/v8.6/ssreflect.ml4 .; $LN ./plugin/v8.6/ssrbool.v ./; $LN ./plugin/v8.6/ssrfun.v ./; $LN ./plugin/v8.6/ssreflect.v ./; MLLIB=ssreflect_plugin.mlpack; EXTRA="ssreflect.ml4"; ;; esac; /home/r/.opam/coq-8.6/bin//coq_makefile -f Make $MLLIB $EXTRA -o Makefile.coq)
Generating Makefile.coq for Coq v8.6 with COQBIN=/home/r/.opam/coq-8.6/bin//
# Override COQDEP to find only the "right" copy .ml files
MAKEFLAGS= --trace make --no-print-directory \
        -f Makefile.coq all \
        COQDEP='/home/r/.opam/coq-8.6/bin//coqdep -exclude-dir plugin -c'
/bin/sh: 1: --trace: not found
Makefile:24: recipe for target 'all' failed
make: *** [all] Error 127
```

--- a/mathcomp/Makefile
+++ b/mathcomp/Makefile
@@ -23,11 +23,11 @@ MAKEFLAGS+=-B
 %:
 	$(H)[ -e Makefile.coq ] || $(call coqmakefile,ssreflect)
 	# Override COQDEP to find only the "right" copy .ml files
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -exclude-dir plugin -c'
 
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq

--- a/mathcomp/algebra/Makefile
+++ b/mathcomp/algebra/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/all/Makefile
+++ b/mathcomp/all/Makefile
@@ -11,12 +11,12 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $*
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/attic/Makefile
+++ b/mathcomp/attic/Makefile
@@ -11,12 +11,12 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $*
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/character/Makefile
+++ b/mathcomp/character/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/field/Makefile
+++ b/mathcomp/field/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/fingroup/Makefile
+++ b/mathcomp/fingroup/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/odd_order/Makefile
+++ b/mathcomp/odd_order/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/real_closed/Makefile
+++ b/mathcomp/real_closed/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/solvable/Makefile
+++ b/mathcomp/solvable/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

--- a/mathcomp/ssreflect/Makefile
+++ b/mathcomp/ssreflect/Makefile
@@ -23,11 +23,11 @@ MAKEFLAGS+=-B
 %:
 	$(H)[ -e Makefile.coq ] || $(call coqmakefile,.)
 	# Override COQDEP to find only the "right" copy .ml files
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -exclude-dir plugin -c'
 
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq

--- a/mathcomp/ssrtest/Makefile
+++ b/mathcomp/ssrtest/Makefile
@@ -30,13 +30,13 @@ MAKEFLAGS+=-B
 
 %:
 	$(H)[ -e Makefile.coq ] || $(COQBIN)/coq_makefile -f Make -o Makefile.coq
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq $* \
 		COQDEP='$(COQDEP) -c'
 
 .PHONY: clean
 clean:
-	$(H)MAKEFLAGS=$(OLD_MAKEFLAGS) $(MAKE) --no-print-directory \
+	$(H)MAKEFLAGS="$(OLD_MAKEFLAGS)" $(MAKE) --no-print-directory \
 		-f Makefile.coq clean
 	$(H)rm -f Makefile.coq
 

